src/World.cpp

```diff
@@ -3502,6 +3500,7 @@ cRedstoneSimulator * cWorld::InitializeRedstoneSimulator(cIniFile & a_IniFile)
 
 	if (NoCaseCompare(SimulatorName, "Incremental") == 0)
 	{
+		// res = new cIncrementalRedstoneSimulator(*this);
```

Comment:
What's this? A commented-out line followed by the same line? Some debugging leftovers?